<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2017 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>addressMerge</class>
 <widget class="QWidget" name="addressMerge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Address Merge Utility</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_process">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Merge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_close">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="_addrTab">
      <attribute name="title">
       <string>Addresses</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="ParameterWidget" name="_filter">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>250</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QPushButton" name="_query">
             <property name="text">
              <string>Query</string>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Maximum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="_searchLit">
           <property name="text">
            <string>Search for:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="XLineEdit" name="_search"/>
         </item>
         <item>
          <widget class="QGroupBox" name="_searchGroup">
           <property name="title">
            <string>Search Through</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="XCheckBox" name="_searchStreet">
              <property name="text">
               <string>Street</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="forgetful">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="XCheckBox" name="_searchCity">
              <property name="text">
               <string>City</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="forgetful">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="XCheckBox" name="_searchPostal">
              <property name="text">
               <string>Postal Code</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="forgetful">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Addresses</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="_frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="1" column="0">
             <widget class="XTreeWidget" name="_addr">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>2</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Right click for selection options</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_selTab">
      <attribute name="title">
       <string>Selections</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QGroupBox" name="_sourceGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Source Addresses</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="XTreeWidget" name="_srcaddr">
            <property name="toolTip">
             <string>Right click for selection options</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="_targetGroup">
         <property name="title">
          <string>Target Address</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="AddressCluster" name="_target">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AddressCluster</class>
   <extends>QWidget</extends>
   <header>addresscluster.h</header>
  </customwidget>
  <customwidget>
   <class>ParameterWidget</class>
   <extends>QWidget</extends>
   <header>parameterwidget.h</header>
  </customwidget>
  <customwidget>
   <class>XCheckBox</class>
   <extends>QCheckBox</extends>
   <header>xcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>XLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
